doc/source/reference/development/howtoadd.rst

```diff
@@ -54,7 +54,7 @@ To keep modules well-organized, **freud** implements the following structure:
 - Python code is stored in the ``freud`` folder at the root of the repository.
 - C++ code is exposed to Python using Cython code contained in pxd files with the following convention: ``freud/_MODULENAME.pxd`` (note the preceding underscore).
 - The core Cython code for modules is contained in ``freud/MODULENAME.pyx`` (no underscore).
-- Generated Cython C++ code (e.g. ``freud/MODULENAME.cpp``) should not be committed during development. These files are updated only at release time.
+- Generated Cython C++ code (e.g. ``freud/MODULENAME.cpp``) should not be committed during development. These files are generated using Cython when building from source, and are unnecessary when installing compiled binaries.
```

Comment:
Are these actually named `*.cxx` now? I noticed that in the `.gitignore`.
```suggestion
- Generated Cython C++ code (e.g. ``freud/MODULENAME.cxx``) should not be committed during development. These files are generated using Cython when building from source, and are unnecessary when installing compiled binaries.
```

Reply:
Yep that's the default output now.

Reply:
@vyasr Did you mean to accept my suggestion? You seemed to agree with my comment, which I think would indicate that we need to change `cpp` to `cxx` in this line about generated code, but you resolved the thread without applying the suggestion.

Reply:
Yes, I must have missed clicking it when I went through to batch the suggestions.